python/fbprophet/forecaster.py

```diff
@@ -199,7 +216,7 @@ def validate_column_name(self, name, check_holidays=True,
             raise ValueError('Name cannot contain "_delim_"')
         reserved_names = [
             'trend', 'additive_terms', 'daily', 'weekly', 'yearly',
-            'holidays', 'zeros', 'extra_regressors_additive','yhat',
+            'holidays', 'zeros', 'extra_regressors_additive', 'yhat',
             'extra_regressors_multiplicative', 'multiplicative_terms',
         ]
         rn_l = [n + '_lower' for n in reserved_names]
```

Comment:
Down in L231, we should pull out all of the holiday names from the countries in append_holidays to be sure that we don't add seasonalities or extra regressors with names that conflict with holidays that will be added.